2006-03-27  Denis Barbier  <barbier@linuxfr.org>

	* sysdeps/unix/sysv/linux/mips/bits/resource.h (RLIMIT_RTPRIO): Fix
	typo.

diff -Nurd glibc-2.4/ports/sysdeps/unix/sysv/linux/mips/bits/resource.h glibc-2.4/ports/sysdeps/unix/sysv/linux/mips/bits/resource.h
--- glibc-2.4/ports/sysdeps/unix/sysv/linux/mips/bits/resource.h	2005-06-20 17:07:34.000000000 +0200
+++ glibc-2.4/ports/sysdeps/unix/sysv/linux/mips/bits/resource.h	2006-08-06 09:40:21.000000000 +0200
@@ -98,7 +98,7 @@
   /* Maximum realtime priority allowed for non-priviledged
      processes.  */
   __RLIMIT_RTPRIO = 14,
-#define RLIMIT_RTPRIO _RLIMIT_RTPRIO
+#define RLIMIT_RTPRIO __RLIMIT_RTPRIO
 
   __RLIMIT_NLIMITS = 15,
   __RLIM_NLIMITS = __RLIMIT_NLIMITS
